app/styles/app.scss

```diff
@@ -19,94 +19,96 @@
  */
 
 
-// VARIABLES
-@import "appuniversum/s-colors.scss";
-@import "appuniversum/s-global";
-@import "appuniversum/s-utilities";
+// EMBER-APPUNIVERSUM
+@import "ember-appuniversum";
```

Comment:
Since this also imports some project specific components and utils I would go with the [more flexible imports](https://appuniversum.github.io/ember-appuniversum/?path=/story/outline-getting-started--page) instead. Example of how this looks in [loket](https://github.com/lblod/frontend-loket/blob/development/app/styles/app.scss).

```scss
@import "ember-appuniversum/a-settings";
@import "ember-appuniversum/a-tools";
@import "ember-appuniversum/a-generic";
@import "ember-appuniversum/a-elements";
@import "ember-appuniversum/a-objects";
@import "ember-appuniversum/a-components";
@import "ember-appuniversum/a-plugins";
@import "ember-appuniversum/a-utilities";
```

That way the util styles can override the custom component styles which the project _might_ depend on at the moment.

Reply:
And I would remove the commented code since it doesn't add much value.

Reply:
I added the flexible imports and the AuApp wrapper.

Reply:
Cool, I would have went with the slightly less flexible "grouped" imports, but this works as well 👍.